common_dwp_api_test/src/main/java/uk/gov/dhsc/htbhf/dwp/testhelper/DWPEligibilityRequestTestDataFactory.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.dhsc.htbhf.dwp.testhelper;
+
+import uk.gov.dhsc.htbhf.dwp.model.DWPEligibilityRequest;
+import uk.gov.dhsc.htbhf.dwp.model.PersonDTO;
+
+import java.time.LocalDate;
+
+import static uk.gov.dhsc.htbhf.TestConstants.UC_MONTHLY_INCOME_THRESHOLD_IN_PENCE;
+
+
+public class DWPEligibilityRequestTestDataFactory {
```

Comment:
Combining a  rename and a package move seems to have caused this class to appear as a new file.